### ISSUE TITLE
[Use of default rules as external ruleset] Number of rules applied does not match those of a normal run

### ISSUE THREAD

**Issue by Beuthmaster** [Bearer/bearer-rules#376]
## Description & Reproduction

To make my CI/CD pipeline independent of external resources, I want to use the rules in this repo as external rules in Bearer.
To achieve this, I downloaded the rules from the rules directory and modified my bearer.yml file accordingly.
When executed, the code analysis runs smoothly, but it applies fewer rules than usual when the rulesets are fetched.

I executed with `bearer scan .  `.
My bearer.yml:
```yaml
disable-version-check: false
log-level: info
report:
    fail-on-severity: critical,high,medium,low
    format: ""
    no-color: false
    output: ""
    report: security
    severity: critical,high,medium,low,warning
rule:
    disable-default-rules: true
    only-rule: []
    skip-rule: []
scan:
    context: ""
    data_subject_mapping: ""
    disable-domain-resolution: true
    domain-resolution-timeout: 3s
    exit-code: -1
    external-rule-dir: ['./rules/python/shared/lang', './rules/python/lang', './rules/ruby/rails', './rules/ruby/shared/rails', './rules/ruby/shared/common', './rules/ruby/shared/aws_lambda', './rules/ruby/shared/lang', './rules/ruby/lang', './rules/ruby/third_parties', './rules/javascript/node', './rules/javascript/react', './rules/javascript/shared/common', './rules/javascript/shared/express', './rules/javascript/shared/aws_lambda', './rules/javascript/shared/lang', './rules/javascript/shared/third_parties', './rules/javascript/express', './rules/javascript/lang', './rules/javascript/third_parties', './rules/javascript/hapi']
    force: true
    hide_progress_bar: false
    internal-domains: []
    parallel: 0
    quiet: false
    scanner:
        - sast
    skip-path: []
    skip-test: true
```
## Expected Behavior

I expected the typical number of rules to be applied:
![image](https://github.com/Bearer/bearer-rules/assets/57135722/5a9ccbf4-753b-41af-a781-41d44f2a65d5)

## Actual Behavior
Less rules are applied and I'm also a bit confused about why my external rules are declared as default rules.
![image](https://github.com/Bearer/bearer-rules/assets/57135722/ace19d11-2408-4970-bff6-0dca0d578e69)

## Your Environment

I'm using a fork of the Bearer Docker Image.
- Output of 'bearer version': 
```
bearer version: 1.43.2, build 1bccddcd46e1476ce31476820e12e6f4b2405799
```

**Comment by cfabianski** [comment]
Thanks @Beuthmaster for your report.
Can you try running it with only `--external-rule-dir=./rules` please? 

> I'm also a bit confused about why my external rules are declared as default rules.

That's because they include a `documentation_url`. Because they include a link, we consider those as default rules.

Let me know

**Comment by Beuthmaster** [comment]
Nice fix, thanks a lot. After having moved all rules into one directory, it runs as expected.
